src/Microsoft.Extensions.Configuration.CommandLine/CommandLineConfigurationExtensions.cs

```diff
@@ -6,13 +6,29 @@
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Extension methods for registering <see cref="CommandLineConfigurationProvider"/> with <see cref="IConfigurationBuilder"/>.
+    /// </summary>
     public static class CommandLineConfigurationExtensions
     {
+        /// <summary>
+        /// Adds an <see cref="IConfigurationProvider"/> that reads configuration values from the command line.
+        /// </summary>
+        /// <param name="configurationBuilder">The <see cref="IConfigurationBuilder"/> to add to.</param>
+        /// <param name="args">The command line args.</param>
```

Comment:
The original bug #120 calls out specifically that the documentation is lacking information on the format supported of command line args. My guess is that at this point we should try to solve that outside of XML docs and in actual documentation. Agreed?